Jenkinsfile

```diff
@@ -135,34 +135,34 @@ pipeline {
         stage('Jtest: Quality Scan') {
             when {
                 expression {
-                    return true;
+                    return false;
```

Comment:
This should have stayed 'true' after you finished with all of your testing where 'false' was convenient to make the pipeline go faster.